name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/eurostathicp-labels_de.properties

```diff
@@ -13,6 +13,8 @@ region.EL = Griechenland
 region.ES = Spanien
 region.FI = Finnland
 region.FR = Frankreich
+region.GB = Gro\u00DFbritannien
+region.GR = Griechenland
```

Comment:
If you look at the data set, the Greece has the code **EL**

http://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=prc_hicp_midx&lang=en

I think it was an error that slipped in with the Spanish translation (which you have fixed below)